apps/desktop-wallet/src/features/balancesOverview/TotalAlphBalance.tsx

```diff
@@ -44,19 +44,19 @@ const TotalAlphBalance = ({ className, type }: TotalAlphBalanceProps) => {
 }
 
 const AvailableAlphAmount = () => {
-  const { totalBalance, totalLockedBalance } = useTotalAlphBalances()
+  const { data } = useAddressesAlphBalances()
 
-  return <AlphBalance balance={totalBalance - totalLockedBalance} />
+  return <AlphBalance balance={data.balance - data.lockedBalance} />
 }
 
 const LockedAlphAmount = () => {
-  const { totalLockedBalance } = useTotalAlphBalances()
+  const { data } = useAddressesAlphBalances()
```

Comment:
Nit: I prefer to name the output variable here, so that it's easy to understand what is feeding the component.

Reply:
I thought about that too and that's how I had originally named them. But I changed my mind after spending some time on inspecting the rerenders. Leaving it as `data` makes it clearer IMO that this object is structurally shared and that it comes directly from Tanstack. Let's talk about it more tomorrow.

Reply:
What about components using multiple query hooks, hence having multiple `data` objects?
But I get you point. We could use a "Data" suffix in the variable names? Like `alphBalancesData`.

Reply:
This is as close as to what the `useQuery`/`useQueries` function returns. That's why I like this pattern.

Reply:
Ah but that's exactly what I was saying 😅 
I also like it when I see `{ data: ... }` and I was suggesting that we use 

```ts
const { data: alphBalances } = useAddressesAlphBalances()
```

Reply:
awesome, yes! 🔥